chart/templates/workspace-api/deployment.yaml

```diff
@@ -102,11 +96,13 @@ spec:
               value: {{ include "carto.postgresql.adminUser" . }}
             - name: POSTGRES_LOGIN_USER
               value: {{ include "carto.postgresql.internalAdminUser" . }}
+            {{- if .Values.internalPostgresql.enabled }}
             - name: POSTGRES_ADMIN_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: {{ include "carto.postgresql.secretName" . }}
                   key: {{ include "carto.postgresql.secret.adminKey" . }}
+            {{- end }}
```

Comment:
That's needed because due to the latest changes we did to ask for just one user the install was failing when the `adminKey` did not have a value :)